### ISSUE TITLE
Empty line between blocks: Should media-queries be considered as block? 

### ISSUE THREAD

**Issue by tzi** [sasstools/sass-lint#780]
Hi!

I'm using sass-lint 1.8.2 with `empty-line-between-blocks: 2`.

I'm surprised that the following code is considered as valid:

```css
.foo {
  content: 'foo';
  @media (min-width: 600px) {
    content: 'bar';
  }
  @include a-mixin-with-content-argument() {
    content: 'baz';
  }
}
```

Is media-queries (and mixin with content argument) shouldn't be considered as block?

I can try to propose a PR if it is validate by the community :wink: 

Cheers,
Thomas.

**Comment by DanPurdy** [comment]
Yeah, this looks like a bug to me. Should definitely be considered a block. Happy to receive a PR to fix this.
